CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 
 * Make Cookie Banner Implementation More Like Design System Implementation [(PR #3325)](https://github.com/alphagov/govuk_publishing_components/pull/3325)
 * Remove capitalisation on GA4 action property value [PR #3367](https://github.com/alphagov/govuk_publishing_components/pull/3367)
+* Add Welsh translation for "Sharing will open the page in a new tab" ([PR #3370](https://github.com/alphagov/govuk_publishing_components/pull/3370))
```

Comment:
Nitpick: can this match the PR title?

Reply:
Actually not sure this warrants a changelog entry, it's such a small thing. Maybe just remove this?

Reply:
I'll update it to match the title, as it does ever so slightly change the user facing output.